src/Arcus.Scripting.LogicApps/Scripts/Cancel-AzLogicAppRuns.ps1

```diff
@@ -1,10 +1,15 @@
 param(
     [Parameter(Mandatory = $true)][string] $ResourceGroupName = $(throw "Name of the resource group is required"),
-    [Parameter(Mandatory = $true)][string] $LogicAppName = $(throw "Name of the logic app is required")
+    [Parameter(Mandatory = $true)][string] $LogicAppName = $(throw "Name of the logic app is required"),
+    [Parameter(Mandatory = $false)][int] $MaximumFollowNextPageLink
 )
 
 try {
-    $runs = Get-AzLogicAppRunHistory -ResourceGroupName $ResourceGroupName -Name $LogicAppName -FollowNextPageLink | 
+    if (!$maximumFollowNextPageLink) {
+        $maximumFollowNextPageLink = 10
```

Comment:
Is this default also something we can place directly in the `param` block, no?

Reply:
You are right, see [`397cc66` (#396)](https://github.com/arcus-azure/arcus.scripting/pull/396/commits/397cc665970f3f3b79cf1dc2b440c07511bbad7e)